tests/FamilyHubs.ServiceDirectory.Api.AcceptanceTests/Tests/Stories/ServiceSearchTests.cs

```diff
@@ -6,28 +6,91 @@ namespace FamilyHubs.ServiceDirectory.Api.AcceptanceTests.Tests.Stories;
 
 // Define the story/feature being tested
 [Story(
-    AsA = "user of the metrics api",
-    IWant = "to be able to update usage information for postcode searches and filters",
-    SoThat = "I can record how search functionality is being used")]
+   AsA = "user of the metrics api",
+   IWant = "to be able to update usage information for postcode searches and filters",
+   SoThat = "I can record how search functionality is being used")]
 
 [TestClass]
 public class ServiceSearchTests
 {
-    private readonly ServiceSearchSteps steps;
+   private readonly ServiceSearchSteps steps;
 
-    public ServiceSearchTests()
-    {
-        //Get instances of the steps required for the test
-        steps = new ServiceSearchSteps();
-    }
+   public ServiceSearchTests()
```

Comment:
nitpick: Since this constructor is only initialising the one variable, you could remove it completely and just have your `steps` declaration as `private readonly ServiceSearchSteps _steps = new();`, but it's a personal preference :)